src/styles/blocks/header.scss

```diff
@@ -0,0 +1,88 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: white;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.logo {
+  display: block;
+  padding: 10px 0;
+  height: 40px;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+  }
+
+  &__items {
+    display: block;
+  }
+
+  &__link {
+    display: block;
+    margin: 0 0 0 20px;
+    height: 60px;
+    line-height: 60px;
```

Comment:
1. it's not a good practice to hardcode height for elements with content inside. Just use padding
2. set a correct line-height from figma